src/swagger-specs/schema-registry.yaml

```diff
@@ -3,28 +3,30 @@ info:
   version: '1.0'
   title: Schema Registry API
   description: >- 
-    The Schema Registry API is used to access the Schema Library within Adobe Experience Platform. The registry provides a user interface and RESTful API from which all available library resources are accessible.
+    The Schema Registry API is used to access the Schema Library within Adobe Experience Platform. The registry provides a user interface and RESTful API from which all available library resources are accessible. 
+    Programmatically manage all schemas and related Experience Data Model (XDM) resources available to you within Platform. This includes those defined by Adobe, Experience Platform partners, and vendors whose applications you use.
```

Comment:
This one can probably have the user-centric "Use the Schema Registry API..." structure that you have for the others